testing/jormungandr-scenario-tests/src/test/non_functional/disruption.rs

```diff
@@ -137,6 +137,9 @@ pub fn passive_leader_disruption_overlap(
     // 2. Only leader is up
     passive.shutdown()?;
 
+    // Wait a bit so that the leader can indeed notice that passive is offline
+    utils::wait(15);
```

Comment:
This is quite a bit of wait...

Reply:
I would rather wait a bit more and avoid spurious failures than do as it is now where the small wait is useless and it's not testing what it is supposed to test

Reply:
Agree with @Zeegomo here. We are not loosing to much time and `cargo test` runs everything in parallel anyway.